README.md

```diff
@@ -25,7 +25,7 @@ Usage of ./observatorium:
     	The log filtering level. Options: 'error', 'warn', 'info', 'debug'. (default "info")
   -metrics.query.endpoint string
     	The endpoint against which to query for metrics.
-  -metrics.write.endpoint string
+  -metrics.receive.endpoint string
```

Comment:
My assumption is that eventually we will have this read from a database as we want to have hard tenancy on the ingestion path, but I think we're fine with this until we actually have that need.